packages/block-editor/src/components/list-view/use-list-view-client-ids.js

```diff
@@ -10,22 +10,29 @@ import { useSelect } from '@wordpress/data';
 import { store as blockEditorStore } from '../../store';
 import { unlock } from '../../lock-unlock';
 
-export default function useListViewClientIds( { blocks, rootClientId } ) {
+export default function useListViewClientIds( {
+	blocks,
+	rootClientId,
+	ignoreRenderingMode,
```

Comment:
Rather than adding this prop, you could instead pass in the `getClientIdsTree` blocks via the `blocks` prop

Reply:
Looks like doing that would end up deferring to `getEnabledClientIdsTreeUnmemoized`. That selector will include the block if its `blockEditingMode` is _not_ `disabled`. 

It's becoming difficult to know where the source of truth is for when a block should/shouldn't be shown.

I agree with @talldan that this prop should be avoided.

Reply:
what @getdave says :) I tried to shape `getEnabledClientIdsTreeUnmemoized` to sometimes show all blocks but making a flag is far easier. The problem is the implementation of content only is very _certain_ of itself and this PR fights it!

Reply:
> Looks like doing that would end up deferring to getEnabledClientIdsTreeUnmemoized.

I might be misunderstanding something, so you'd need to explain it more. Or my suggestion is being misinterpreted. To clarify - I'm suggesting doing this:
```jsx
const clientIdsTree = useSelect( ( select ) => getClientIdsTree( rootClientId ), [ rootClientId ] );
return (
    <ListView blocks={ clientIdsTree } />
);
```

And it looks like ListView would use the correct blocks because it has this internally:
```js
clientIdsTree: blocks ?? getEnabledClientIdsTree( rootClientId ),
```

Having said that, I don't understand why you want to circumvent `blockEditingMode`, that doesn't feel right to me. It'll lead to having to also circumvent the block editing mode of the controlled inner blocks so that they can be selected/edited. Can't the blocks have the correct block editing mode?

Reply:
> I don't understand why you want to circumvent blockEditingMode, that doesn't feel right to me. It'll lead to having to also circumvent the block editing mode of the controlled inner blocks so that they can be selected/edited. Can't the blocks have the correct block editing mode?

This is the gist of what's blocking this PR and the Issue for a solution! The problem is we're fighting content only because we want a special editing UX for blocks that do not declare content role attributes to only sometimes be edtitable in certain contexts. :/